Tests/FoundationInternationalizationTests/CalendarTests.swift

```diff
@@ -884,7 +922,6 @@ final class CalendarBridgingTests : XCTestCase {
 #endif
 
 
-#if ENABLE_CALENDAR_COMPATIBILITY_TEST // These tests take a long time to run, so disable them for now
```

Comment:
Just want to make sure - enabling these tests was intentional right?

Reply:
Yes! Enabling them for CI to test the changes. I'm disabling them in a follow up PR